plugins/binanceExchange_ws.go

```diff
@@ -277,10 +277,26 @@ func (beWs *binanceExchangeWs) subscribeStream(symbol, format string, subscribe
 	beWs.streams[streamName] = stream
 	beWs.streamLock.Unlock()
 
+	timePassed := time.Duration(0)
+	checkTime := time.Millisecond * 50
+
 	//Wait for binance to send events
-	time.Sleep(timeToWaitForFirstEvent)
 
-	data, isStream := state.Get(symbol)
+	var (
```

Comment:
I don't think we need this and it's making the code more complex than we need.
I will later add logic to initialize all required streams so we won't face this issue at runtime in the future. Best to keep this simple for now